pontoon/administration/views.py

```diff
@@ -196,11 +213,13 @@ def manage_project(request, slug=None, template='admin_project.html'):
         'form': form,
         'subpage_formset': subpage_formset,
         'repo_formset': repo_formset,
+        'tag_formset': tag_formset,
         'external_resource_formset': external_resource_formset,
         'locales_selected': locales_selected,
         'locales_available': Locale.objects.exclude(pk__in=locales_selected),
         'subtitle': subtitle,
         'pk': pk,
+        'project': project,
```

Comment:
It seems like we don't need that.

Reply:
ill check and remove...

Reply:
it turns out its there for later PR - leaving in as it will be used quite soon